README.md

```diff
@@ -209,6 +209,45 @@ Methods that are mapped using `map_motion` accept an `event` parameter which is
 
 See the code for full API for [Event](https://github.com/unabridged/motion/blob/master/lib/motion/event.rb) and [Element](https://github.com/unabridged/motion/blob/master/lib/motion/element.rb).
 
+## Callbacks
+
+Motion has callbacks which will let you pass data from a child component back up to the parent. `bind` is how Motion figures out what method to call. `on_click` is a method passed into the child component and can be named whatever you'd like. When the component's motion is called, the motion calls the method in the parent component.
```

Comment:
I'm not sure about calling out `on_click` specifically like this or calling it a method. The name is chosen by the user and it is a `Motion::Callback` object.

I would explain it more like this:

> Sometimes you want to pass information from a component back up to one of its parents. In Motion, this can be  accomplished with callbacks. Callbacks are created by calling `bind` with the name of a method on the component which will act as a handler. It returns a new callback which can be passed to child components like any other state. To invoke the handler from the callback, use `call`. If the handler accepts an argument, it will receive anything that is passed to `call`.

Reply:
How about
> Motion has callbacks which will let you pass data from a child component back up to the parent. Callbacks are created by calling `bind` with the name of a method on the parent component which will act as a handler. It returns a new callback which can be passed to child components like any other state. To invoke the handler from the callback, use `call`. If the handler accepts an argument, it will receive anything that is passed to `call`.

this? Basically my first sentence and then the rest of yours, with a clarification on which component the method is on.

Reply:
That looks good to me! :+1: